cleverpush-example/src/main/java/com/example/cleverpush/MyNotificationExtenderService.java

```diff
@@ -10,15 +9,11 @@ public class MyNotificationExtenderService extends NotificationExtenderService {
     @Override
     protected boolean onNotificationProcessing(Notification notification) {
         // modify notification
-        notification.setExtender(new NotificationCompat.Extender() {
-            @Override
-            public NotificationCompat.Builder extend(NotificationCompat.Builder builder) {
-                builder.setColor(new BigInteger("FF00FF00", 16).intValue()); // Set notification color to green
-                return builder;
-            }
+        notification.setExtender(builder -> {
+            builder.setColor(new BigInteger("FF00FF00", 16).intValue()); // Set notification color to green
+            return builder;
         });
 
-        // return true to not display notification
```

Comment:
why remove this comment?

Reply:
okay I have changed AppActivity to Activity